java-filter-chain/src/main/java/org/wso2/mgw/filterchain/JWTValidator/JWTValidator.java

```diff
@@ -33,14 +33,43 @@
 import java.util.Base64;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
 import com.nimbusds.jose.JOSEException;
 import com.nimbusds.jose.JWSHeader;
 import com.nimbusds.jwt.JWTClaimsSet;
 import com.nimbusds.jwt.JWTParser;
 import com.nimbusds.jwt.SignedJWT;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
 
 public class JWTValidator{
     private static RSAPublicKey publicKey = readPublicKey();
+    private static JWSVerifier jwsVerifier = new RSASSAVerifier(publicKey);
+    private static boolean enableCache = true;
```

Comment:
Lets read this from environment variable, so we can run perf without code changes for cache enable and disable cases